src/main/java/teammates/ui/template/ArchivedCoursesTable.java

```diff
@@ -0,0 +1,28 @@
+package teammates.ui.template;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArchivedCoursesTable {
+    private List<ArchivedCoursesTableRow> rows;
+
+    public ArchivedCoursesTable(){
+        rows = new ArrayList<ArchivedCoursesTableRow>();
+    }
+    
+    public List<ArchivedCoursesTableRow> getRows() {
+        return rows;
+    }
+
+    public void setRows(List<ArchivedCoursesTableRow> rows) {
+        this.rows = rows;
+    }
+
+    public int getSize() {
+        if (rows == null) {
+            return 0;
+        }
+        return rows.size();
+    }
```

Comment:
I believe getRows() combined with a .size() or .isEmpty() should be enough for most cases on the Java side.
If this is needed on the tags / JSP, there are also equivalent functions in the `fn` library.